File: 1046 - Tempo de jogo.c
#include <stdio.h>

int main(void)
{
    int n1, n2;
    scanf(" %d %d", &n1, &n2);
    
    if (n1 >= n2) {
        n2 += 24;
        n2 -= n1;
        
    }else{
        n2 -= n1;
    }
    printf("O JOGO DUROU %d HORA(S)\n", n2);
    
}

File: Python/Lista 3/1046 - Tempo de Jogo.py
temp = input()
temp = temp.split()

start = int(temp[0])
end = int(temp[1])

if (start == end):
    print("O JOGO DUROU 24 HORA(S)")

else:
    end = end + 24 if (end < start) else end

    time = end - start
    print("O JOGO DUROU {} HORA(S)".format(time))
